src/da/activate.ts

```diff
@@ -0,0 +1,126 @@
+import * as vscode from "vscode";
+
+import { regalPath } from "./../ls/clients/regal";
+import { promptForUpdateRegal } from "./../ls/clients/regal";
+
+const minimumSupportedRegalVersion = "0.26.0";
+
+export function activateDebugger(context: vscode.ExtensionContext) {
+  context.subscriptions.push(
+    vscode.commands.registerCommand("opa.debug.debugWorkspace", (resource: vscode.Uri) => {
+      let targetResource = resource;
+      if (!targetResource && vscode.window.activeTextEditor) {
+        targetResource = vscode.window.activeTextEditor.document.uri;
+      }
+      if (targetResource) {
+        vscode.debug.startDebugging(undefined, {
+          type: "opa-debug",
+          name: "Debug Workspace",
+          request: "launch",
+          command: "eval",
+          query: "data",
+          bundlePaths: ["${workspaceFolder}"],
+          enablePrint: true,
```

Comment:
Shouldn't these values be fetched from the config? Or that's not been resolved at this point?

Same question applies below where similar objects are returned.

Reply:
The below objects are here to prime new launch configurations in `.vscode/launch.json` with default values.

This specific object is for when you launch an ad-hoc debug session with the debug button in the top right of the editor. I suppose we could allow you to configure this through `package.json`. But on the other hand, if you're not happy with the default values, you can just create a custom, names launch config in `.vscode/launch.json`.

Reply:
Just trying to ensure that we set as smart defaults as possible, and if you've already set e.g. bundlePaths in one location (your `.vscode/settings.json`) you shouldn't have to do it twice.

Reply:
I think if I drop `bundlePaths` here, it should pan out to defaulting to whatever value you've set for `opa.roots` (once that change is in).